client/product_details/AddToBag.jsx

```diff
@@ -39,30 +44,69 @@ class AddToBag extends React.Component {
     });
   }
   
-  handleQuantitySubmit(e) {
+  handleQuantityChange(e) {
     var selectedQuantity = e.target.value;
     this.setState({
       selectedQuantity: selectedQuantity
     });
   }
 
+  handleAddToBagSubmit() {
+    if (this.state.selectedSize !== 'SELECT SIZE') {
+      for (var i = 0; i < this.props.availableSizes.length; i++) {
+        if (this.props.availableSizes[i][0] === this.state.selectedSize) {
+          var sku_id = this.props.availableSizes[i][2];
+          console.log('styleId: ', sku_id);
```

Comment:
Hi @alizehrehman, just a minor thing but I'm wondering if you plan to remove this console.log at some point? I think it's totally okay for now though especially if you're still working on this piece.